src/main/java/com/karimbouchareb/chromatographysimulator/ui/SplashScreen.java

```diff
@@ -0,0 +1,206 @@
+package com.karimbouchareb.chromatographysimulator.ui;
+
+import com.karimbouchareb.chromatographysimulator.ChromatographySimulatorApp;
+import javafx.animation.FadeTransition;
+import javafx.animation.RotateTransition;
+import javafx.animation.TranslateTransition;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+import javafx.util.Duration;
+import org.kordamp.ikonli.javafx.FontIcon;
+import org.kordamp.ikonli.materialdesign.MaterialDesign;
+
+import static com.karimbouchareb.chromatographysimulator.ChromatographySimulatorApp.SCREEN_BOUNDS;
+import static com.karimbouchareb.chromatographysimulator.ChromatographySimulatorApp.makeImageView;
+
+public class SplashScreen {
```

Comment:
First thing I notice in this PR is that there is no red - no code being subtracted. Was this code new or duplicated from ChromatographySimulatorApp.java?